### PR TITLE
push fogapi updates

### PR DESCRIPTION
**Pull request by darksidemilk**
Updates to the fogapi module at the git level. The newest version has been published to PSGallery

### CODE REVIEW

File: PowershellModules/FogApi/FogApi/FogApi.psd1 [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+GUID = '7aa922fa-bb4f-46a0-a478-684e9535c65d'
+
+# Author of this module
+Author = 'JJ Fullmer'
+
+# Company or vendor of this module
+CompanyName = 'FOG Project'
+
+# Copyright statement for this module
+Copyright = '(c) 2018 JJ Fullmer. All rights reserved.'
```
Comment (wayneworkman) [inline PowershellModules/FogApi/FogApi/FogApi.psd1]: Does this mean the fog-community-scripts license does not apply to this module?

**Comment by darksidemilk** [comment]
That's a default message when you create a module manifest, let me double check what I put there last time. I think the license it's published with, which I believe is the mit license, is more important

**Comment by darksidemilk** [comment]
That is what the current version says, but down lower the license linked to is the fog community scripts license. That's just the default message and doesn't restrict usage in anyway in the fog community scripts. I think it's more of I'm the author and I reserve the right to do what I want with my code, and I choose to share it and license it in this community. But I'm no IP lawyer.

**Comment by darksidemilk** [comment]
I just looked at another module I use, jiraPS, it has that same message but is still open source. So, confirmed,  it's nothing to worry about.

**Pull request merged**